cmd/kubeadm/app/cmd/reset.go

```diff
@@ -184,62 +181,20 @@ func (r *Reset) Run(out io.Writer) error {
 	return nil
 }
 
-func reset(execer utilsexec.Interface, dockerCheck preflight.Checker, criSocketPath string) {
-	crictlPath, err := execer.LookPath("crictl")
-	if err == nil {
-		resetWithCrictl(execer, dockerCheck, criSocketPath, crictlPath)
```

Comment:
if I'm reading this right, we used to try to use crictl for removing docker containers earlier (given crictl existed in PATH)
What's changing is that we now don't try to do that, but always use docker for dockershim.sock
The other change is that in case of an other CRI runtime, don't fallback on docker.
Is that right? Both of these changes SGTM

Reply:
Yes, that's correct. I tried to make it more explicit and only use crictl when user specifies CRI socket, otherwise use docker. Theoretically it would probably be even better if we add new command line option for that (--use-cri), but I didn't want to do that in this PR as it's already quite big.